src/device-registry/config/constants.js

```diff
@@ -8,7 +8,7 @@ const devConfig = {
   DB_NAME: process.env.MONGO_DEV,
   REDIS_SERVER: process.env.REDIS_SERVER_DEV,
   REDIS_PORT: process.env.REDIS_PORT,
-  KAFKA_BOOTSTRAP_SERVERS: process.env.KAFKA_BOOTSTRAP_SERVERS_DEV.split(","),
+  KAFKA_BOOTSTRAP_SERVERS: (process.env.KAFKA_BOOTSTRAP_SERVERS_DEV||"").split(","),
```

Comment:
Preventing `"TypeError: Cannot read properties of undefined (reading 'split')"` when I run `npm run dev-mac`
<img src ="https://user-images.githubusercontent.com/86492979/219195724-aea4ad2b-96e6-453f-ada9-b7f6cc9bebb2.png" height =300>

Reply:
Thanks @BenjaminSsempala for this. 

1. I guess you need tot have that ENV in your local ENV?
2. I also go it sometime  about 2 days ago and it just disappeared., I suspect is something to do with Node but we can share notes soon.
3. I also like that approach is using the empty String as a work around. Let us continue to investigate this mysterious behavior.

Thanks again!

Reply:
@BenjaminSsempala : just sharing more insights on this.

I guess this is because those ENVs are loaded when the` entire` APP is run (started).
So if you just run a file within the app without starting the whole of it, we will get such errors --- as expected.

So how do we solve this with our unit testing fundamentals? 

![Uploading Screenshot 2023-02-21 at 10.52.17.png…]()

Reply:
Quick suggestion @BenjaminSsempala ....what if you just load `require('dotenv').config();` at the top of the unit test file?

It has worked after trying it out on my local machine, See attached screenshots for illustration.
<img width="1672" alt="Screenshot 2023-02-21 at 13 17 21" src="https://user-images.githubusercontent.com/1590213/220317213-42e6bfe0-43bc-4501-a32d-71ac6eb7e15e.png">
<img width="944" alt="Screenshot 2023-02-21 at 13 17 52" src="https://user-images.githubusercontent.com/1590213/220317238-ed289b7a-bd2d-4557-b685-b2335eb54f6a.png">

Reply:
> Quick suggestion @BenjaminSsempala ....what if you just load `require('dotenv').config();` at the top of the unit test file?
> 
> It has worked after trying it out on my local machine, See attached screenshots for illustration. 

Thank you @Baalmart , This solution worked for me